test/tests_for_sdkmr/ee2_SDKMethodRunner_test_EE2Runjob_test.py

```diff
@@ -281,6 +465,24 @@ def test_run_job_batch(self, rq_mock, condor_mock):
             jobs = [job, job_bad]
             runner.run_job_batch(params=jobs, batch_params={"wsid": self.ws_id})
 
+        # Squeeze in a retry test here
+        parent_job_id = job_ids["parent_job_id"]
+        child_job_id = job_ids["child_job_ids"][0]
+        runner.update_job_status(job_id=child_job_id, status=Status.terminated.value)
+        parent_job = runner.check_job(job_id=parent_job_id)
+        assert len(parent_job["child_jobs"]) == 3
+        retry_id = runner.retry(job_id=child_job_id)["retry_id"]
+        parent_job = runner.check_job(job_id=parent_job_id)
+        assert len(parent_job["child_jobs"]) == 4
+        assert parent_job["child_jobs"][-1] == retry_id
+
+        # Test to see if one input fails, so fail them all
+        with self.assertRaises(expected_exception=RetryFailureException):
+            retry_id = runner.retry_multiple(
+                job_ids=[child_job_id, child_job_id, "fail"]
+            )
+            print(retry_id)
```

Comment:
Shouldn't the test check that the other jobs weren't retried?

Reply:
Sure, done

Reply:
:+1: